libs/shared/src/consts/providers/credentials/provider-credentials.ts

```diff
@@ -181,7 +181,7 @@ export const nodemailerConfig: IConfigCredentials[] = [
     type: 'string',
     required: false,
   },
-  ...mailConfigBase,
+  mailConfigBase[0],
```

Comment:
I consider we shouldn't remove the option to set the sender name from Nodemailer. If it is not used in the provider package we should implement it. Probably the change for the credentials would be to make the sender name optional instead of mandatory as it is right now.